src/cli.rs

```diff
@@ -264,9 +264,8 @@ pub async fn main<H: Hooks, M: MigratorTrait>() -> eyre::Result<()> {
     let environment: Environment = cli.environment.unwrap_or_else(resolve_from_env).into();
 
     let config = environment.load()?;
-
-    if !H::init_logger(&config, &environment)? {
-        logger::init::<H>(&config.logger);
+    if let Some(l) = config.logger.as_ref() {
```

Comment:
Trying to discuss this for a minute, so we're saying - if there's no logger section, logging is never initialized?

I'm torn about that. Somehow I feel there should always be a logger section and then control it with enable/disable. I don't know why but logging feels like a special case where you always want to see that config visually and then tweak it but never completely remove it from a file.

WDYT?

Reply:
Yeah this is a fair point.

I think this warrants adding some tests as exemplars 😅

Reply:
I've changed this now in c85dde732ab3c7ee0d16498d7b818ad142ffb5f5 to use a default logger config when its either not configured `None` or disabled `Some(Enabled::Off)`.

The fields can still be present in the config file, even when the feature is `Enabled::Off`. 
They are just not required 😅